definitions/tools/star_align_fusion.cwl

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env cwl-runner
+class: CommandLineTool
+cwlVersion: v1.0
+label: "STAR: align reads to transcriptome"
+baseCommand: ["/usr/local/bin/STAR"]
+requirements:
+    - class: ShellCommandRequirement
+    - class: ResourceRequirement
+      ramMin: 64000
+      coresMin: 10
+    - class: DockerRequirement
+      dockerPull: "sridnona/mgirnaseq:v3"
+arguments: [
+    "--runThreadN", $(runtime.cores)
+
+]
+inputs:
+    outSAMtype:
+        type: string[]
+        default: ["BAM", "Unsorted"]
+        inputBinding:
+            position: 3
+            prefix: '--outSAMtype'
+    runMode:
+        type: string
+        default: "alignReads"
+        inputBinding:
+            position: 2
+            prefix: "--runMode"
+        doc: |
+          string: type of the run
+          alignReads - map reads
+          genomeGenerate - generate genome files
```

Comment:
This looks like it could be an `enum` (similar to https://github.com/genome/analysis-workflows/blob/42b4fa00a93da4efe5d98639c5aa666bc8732979/definitions/tools/kallisto.cwl#L35-L38 ).

Reply:
we wont be using this workflow for index generation so `alignReads` set default no enum required IMO